components/Starknet/modules/architecture_and_concepts/pages/Accounts/deploying_new_accounts.adoc

```diff
@@ -1,83 +1,48 @@
 [id="deploying_new_accounts"]
-= Deploying new accounts
+= Deploying a new account
 
-Starknet provides the `deploy_account` transaction in order to deploy new accounts to the
-network.
+You can deploy a new account in the following ways:
 
-== Deploy account transaction
+* Send a `DEPLOY_ACCOUNT` transaction. This method does not require a preexisting account.
+* Include the `deploy` syscall in the account contract, such as by invoking the Universal Deployer Contract (UDC) with an `INVOKE` transaction. This method requires an existing account to send the `INVOKE` transaction.
 
-In order to deploy a new account on StarkNet you need to complete the following steps:
+Upon receiving one of these transactions, the sequencer performs the following steps:
 
-* Decide on the account contract that you want to deploy
-* Compute your would-be account address off-chain
-* Send funds to this address
+. Verifies that the address has funds to pay for the deployment.
```

Comment:
```suggestion
. Verifies that the address has funds to pay for the deployment.
+
If you use a `DEPLOY_ACCOUNT` transaction, the fees are paid from the address of the deployed account. If you use the UDC, the fees are paid from the sender's account.
```

Reply:
Fixed.

Reply:
Remove it from the numbered list and add as a note. For `DEPLOY_ACCOUNT` tx, the fees are charged from the deployed address and from an `INVOKE` tx using the UDC, the fees are charged from the sender's account (and add a link to the transactions types page to invoke tx v1 and v3).

Reply:
Fixed.